addons/l10n_ar/views/account_move_view.xml

```diff
@@ -30,6 +30,10 @@
                     </div>
                 </group>
             </group>
+
+            <xpath expr="//div[@name='journal_div']//field[@name='journal_id']" position="attributes">
+                 <attribute name="attrs">{'readonly': ['|', '&amp;', ('country_code', '=', 'AR'), ('posted_before', '=', True), '&amp;', ('posted_before', '=', True), ('name', 'not in', (False, '', '/'))]}</attribute>
```

Comment:
Had a test that was failing "test_01_create_invoice", this test is failing because when creating the invoice in the setup from the method _create_test_invoices_like_demo, the test_invoice_6 is using a different journal "sale_expo_journal_ri". 
So the test fail because the afip_pos_number is not the right one, the problem is that the afip_pos_number is changed in the _inverse_l10n_latam_document_number which is compute from the name and the name is compute from another field and this field is computed from another thing, from what i've seen the journal at a certain moment is not the right one and so it goes through the if statement to modify the pos_number. 

We came to the conclusion that putting an override for that case was ok but i can still try to find another way to fix the problem but it's a inverse/compute nightmare in there and we couldn't find the a cleaner solution ahah

Reply:
Wait just relaunched the test and it's working :facepalm:

Reply:
I will wait for the l10n build to end before pinging you for the r+, let's see if it's green ahah